Projects/Community-Plugins/Schemas/Docs-Nodes/P/Plugins/Plugins/plugins.json

```diff
@@ -9,14 +9,9 @@
                 "updated": 1628037249031
             },
             {
-                "language": "EL",
-                "text": "Η ιεραρχία των πρόσθετων καθορίζει ποιά πρόσθετα θα συμπεριληφθούν στον τρέχοντα χώρο εργασίας. Στο πλαίσιο αυτής της ιεραρχίας, τα πρόσθετα είναι αρχεία JSON που διαθέτουν την περιγραφή μιας ιεραρχίας όπως ένα εξορυκτήριο δεδομένων, ένα εξορυκτήριο συναλλαγών, μια μηχανή συναλλαγών ή ένα σύστημα συναλλαγών.",
-                "updated": 1641828795260
-            },
-            {
-                "language": "DE",
-                "text": "Die Plugin-Hierarchie definiert, welche Plugins in den aktuellen Arbeitsbereich aufgenommen werden sollen. Im Kontext dieser Hierarchie sind Plugins JSON-Dateien mit der Beschreibung einer Hierarchie, z. B. einer Data Mine, Trading Mine, Trading Engine oder eines Trading Systems.",
-                "updated": 1641764838784
+                "language": "TR",
+                "text": "Eklentiler hiyerarşisi, geçerli çalışma alanına hangi eklentilerin dahil edileceğini tanımlar. Bu hiyerarşi bağlamında eklentiler, veri madeni, ticaret madeni, ticaret motoru veya ticaret sistemi gibi bir hiyerarşiyi tanımlayan JSON dosyalarıdır.",
+                "updated": 1644582475185
```

Comment:
It still shows me that thre spanish and german translation gets overwritten.

Reply:
> It still shows me that thre spanish and german translation gets overwritten.

I don't know how to fix this. what should I do?

Reply:
![image](https://user-images.githubusercontent.com/2537958/155146797-e7cd8a53-0634-4922-84e5-81b03a47b941.png)
It should be like this

Reply:
thanks maxah

Reply:
> It still shows me that thre spanish and german translation gets overwritten.
can you check? is there any error?

Reply:
Save the file in Superalgos\Projects\Community-Plugins\Schemas\Docs-Nodes\P\Plugins\Plugins\plugins.json and run app.contribute
[plugins.txt](https://github.com/Superalgos/Superalgos/files/8119732/plugins.txt)

Reply:
> It still shows me that thre spanish and german translation gets overwritten.

seems to be fixed now